setup.py

```diff
@@ -14,6 +14,7 @@
     author="Valentin Sulzer",
```

Comment:
we should definitely change this now!

Reply:
Perhaps we can say "The PyBaMM development team", and have an AUTHORS file with all the names?

Reply:
yep sounds good, we could even just leave `author` out completely

Reply:
ok, I've now removed the author and author email